sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/models/ChatMessageReceivedEvent.java

```diff
@@ -12,38 +14,45 @@
      * Type of the chat message.
      * The only type currently supported is Text
      */
+    @JsonProperty(value = "type")
```

Comment:
do we want to consider an ext-enum for this?

Reply:
+1 The public facing version of this (if there is a public version) should be extended enum. But if it is purely internal, then I think it is ok, right @anuchandy?

Reply:
yes, it's fine for internal types. It seems this model is a part of public API, the model represents event emitted to the public `on` method

Reply:
Yeah it seems that they are exposed through `ChatEventId` which should become an extended enum.